src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public enum Color {
+    red,
+    blue,
```

Comment:
```suggestion
    RED,
    BLUE,
```
naming convention for enum in Java is different